supervisor/dbus/manager.py

```diff
@@ -36,6 +38,7 @@ def __init__(self, coresys: CoreSys) -> None:
         self._agent: OSAgent = OSAgent()
         self._timedate: TimeDate = TimeDate()
         self._resolved: Resolved = Resolved()
+        self._board: BoardManager = BoardManager()
```

Comment:
that should be part of `self._agent.board` from `OSAgent` not `DbusManager`

Reply:
https://github.com/home-assistant/supervisor/blob/main/supervisor/dbus/agent/__init__.py#L42